.env.example

```diff
@@ -1,6 +1,7 @@
 # YOUR DPL-CMS NEEDS TO BE RUNNING LOCALLY
 NEXT_PUBLIC_GRAPHQL_SCHEMA_ENDPOINT_DPL_CMS=https://dapple-cms.docker/graphql
-NEXT_PUBLIC_GRAPHQL_BASIC_TOKEN_DPL_CMS=Z3JhcGhxbF9jb25zdW1lcjp0ZXN0
+# BASIC AUTH HEADER - YOUR LOCAL CONSUMER KEY AND SECRET FOR DPL-CMS
+NEXT_PUBLIC_GRAPHQL_BASIC_TOKEN_DPL_CMS="Z29fZ3JhcGhxbDpvQThtcVpGeG1GdWJSRUg3N1QzcWVBcGVvVXhvTlM3WQ=="
```

Comment:
Nice. Please create separate PR's for stuff like this in the future. 👍  Because then we can get it into main faster.